### ISSUE TITLE
Finetune DeepSORT hyperparameters, investigate/research better tracking algorithms.

### ISSUE THREAD

**Issue by Pecneb** [Pecneb/computer_vision_research#48]
Improve tracking accuracy through finetuning DeepSORT's hyperparameters; research alternative tracking algorithms.

**Comment by Pecneb** [comment]
```parser.add_argument("--max_cosine_distance", type=float, default=10.0,
                        help="Gating threshold for cosine distance metric (object appearance).")
    parser.add_argument("--max_iou_distance", default=0.7, type=float)```

**Comment by Pecneb** [comment]
`max_age` parameter of the DeepSORT Track Class can be adjusted. This seems to be working. For now `max_age=15` was set. Other tests with other parameter values can be done in the future.

**Comment by Pecneb** [comment]
Masking out regions in the videos helped as well.
